### ISSUE TITLE
More lua based example of usage?

### ISSUE THREAD

**Issue by thepenguinthatwants** [michaelb/sniprun#74]
Hi,
I am trying to switch into lua and your plugin looks very nice. Althought I have yet to get this work correctly.
Could you update your readme.md to include more lua based examples?

Installed using packer, but for now I haven't even gotten python to run.

`use {'michaelb/sniprun', run = 'bash install.sh'}`

Could you add more examples on lua instead of the old syntax?
```
nmap <leader>ff <Plug>SnipRun
nmap <leader>f <Plug>SnipRunOperator
vmap f <Plug>SnipRun
```

**Comment by michaelb** [comment]
That requires going a bit out of my way, but ok.

I want sniprun to stay as noob-friendly as possible, so to not clutter the README and still provide the 'vimscript' config, lua-specific things will probably be collapsed like this

<detail>
some examples
</detail>

Meanwhile, if you figure out how to run that post-install script with packer, you'll save me the trouble of going through the ordeal (i'm not using packer myself) if you can document it here

**Comment by michaelb** [comment]
Fixed in v0.5.4 :-)
